swap/src/monero/wallet.rs

```diff
@@ -330,7 +341,12 @@ where
 
         if tx.confirmations > seen_confirmations {
             seen_confirmations = tx.confirmations;
-            tracing::info!(%txid, "Monero lock tx has {} out of {} confirmations", tx.confirmations, conf_target);
+            tracing::info!(
+                %txid,
+                %seen_confirmations,
+                needed_confirmations = %conf_target,
+                "Monero lock transaction update."
+            );
```

Comment:
Every `info!` within the CLI is user-facing, so these messages should really be handled with care IMO. I am sure we can say something better here other than just "Lock transaction update" which seems to be a pure placeholder so we can log out the values.

Reply:
I think it is good to keep in mind that info (and debug if switched on) are currently user-facing on the CLI but I would not waste too much time into making the log messages more user friendly. IMO the above is fine.

I see the current solution as temporary. Eventually we will add a (simple) UI that processes the JSON and the user won't see the logs anymore.

Reply:
Same as noted above, the issue I see here is that we use a tracing event to dump key-value pairs on the user and provide a message that describes what this information is rather than having the message by useful on its own.

A log message provides more value if it contains a clear _action_ of what happened when this particular line executed.

`Monero lock transaction update.` doesn't actually tell me what changed. Okay there is an update, but an update on what? Is the update good or bad? Should I be waiting for more updates?

Compared that to something like (exact wording subject to bike-shedding, also I don't want specify out complete solutions in my review here):

- `Confirmations for Monero lock tx increased`
- `Received new confirmation for Monero lock tx`

Even without the key-value pairs, this provides me with useful information.

Reply:
Updated to `Received new confirmation for Monero lock tx`